internal/socket/zsyscall_windows.go

```diff
@@ -48,23 +48,23 @@ var (
 )
 
 func bind(s windows.Handle, name unsafe.Pointer, namelen int32) (err error) {
-	r1, _, e1 := syscall.Syscall(procbind.Addr(), 3, uintptr(s), uintptr(name), uintptr(namelen))
+	r1, _, e1 := syscall.SyscallN(procbind.Addr(), uintptr(s), uintptr(name), uintptr(namelen))
```

Comment:
:warning: I should add that I patched these manually (don't think I can use `go generate` unless doing so on a Windows machine).

Not 100% sure if CI here verifies these files (if not, perhaps someone could double-check them and run `go generate` on a Windows machine)

Reply:
> Not 100% sure if CI here verifies these files (if not, perhaps someone could double-check them and run `go generate` on a Windows machine)

looks like it does, and CI looks happy; https://github.com/thaJeztah/go-winio/actions/runs/4645933496/jobs/8222049635